### PR TITLE
Add track descriptions to CFP

### PR DESCRIPTION
**Pull request by RyanSkraba**
As requested and started by @Claudenw, it's useful to have the track descriptions available!

I rewrote some of the texts to be a bit more consistent with each other without changing the meaning, but it's pretty obvious they all have different authors.  I asked permission where I made larger changes.

### CODE REVIEW

File: content/blog/cfp-open.md [hunk 1]
```diff
@@ -45,4 +45,90 @@ All my best, and thanks so much for your participation,
...
+This track welcomes submissions related to innovation and improvements in the storage software space.
+
+**Cassandra**:
+All things Apache Cassandra, focusing on 5.0 features, what's next in 5.1, how we do engineering, and our community.
+
+**CloudStack**:
+Presenting the new Apache CloudStack release, new integrations and capabilities and user stories.
+
+**Community**:
+Community, Governance, and Culture, covering the social and organizational aspects of ASF projects, and the Foundation at large.
```
Comment (Claudenw) [inline content/blog/cfp-open.md]: I think that we should rename the track "Community, Governance, and Culture" 

alternatively the first sentence could read:

"This track covers the  Community, Governance, and Culture, including the social and organizational aspects, of ASF projects and the Foundation at large."

I tend to agree that the "this track" prefix is a bit tired and dated, but with what I perceive as a naming mismatch. I think it is justified here.

File: content/blog/cfp-open.md [hunk 2]
```diff
@@ -45,4 +45,90 @@ All my best, and thanks so much for your participation,
...
+* In-house, managed service and cloud providers of Apache software
+* End-users of Apache Software
+* The wider open-source, developer and research communities.
+
+**Posters**:
+Posters about ASF projects and topics of interest to ASF projects.
+Posters with original research are more likely to be accepted.
+We are looking for ideas and concepts that are applicable across projects.
+Posters do not have to be associated with an ASF project.
+Access/use of all information presented or referenced may not be inhibited by patent or other licensing restrictions.
```
Comment (Claudenw) [inline content/blog/cfp-open.md]: I thnk we should change "may not be inhibited by" to " must be unencumbered by"

**Review by Claudenw — COMMENTED** [review]
This is not a review comment of this page but rather a reminder and suggestion for the sessionize page.  

I added the same data to the sessionize page last night.  I think that we should finish the wordsmithing on this page and then simpley link to it from the sessionize page.  This means removing all the text I put in on the sessionize page yesterday.

File: content/blog/cfp-open.md (hunk 1 above)
Comment (rzo1) [reply content/blog/cfp-open.md]: +1

File: content/blog/cfp-open.md (hunk 2 above)
Comment (rzo1) [reply content/blog/cfp-open.md]: +1

File: content/blog/cfp-open.md (hunk 1 above)
Comment (fpapon) [reply content/blog/cfp-open.md]: +1

File: content/blog/cfp-open.md (hunk 2 above)
Comment (fpapon) [reply content/blog/cfp-open.md]: +1

**Comment by fpapon** [comment]
Do we need to add the track leader names? I remember that they asked to in the mailing list.

**Comment by Claudenw** [comment]
> Do we need to add the track leader names? I remember that they asked to in the mailing list.

I don't think for this public facing page.  We do need to track that for our in-house communication.

File: content/blog/cfp-open.md (hunk 1 above)
Comment (RyanSkraba) [inline content/blog/cfp-open.md]:
```suggestion
This track covers the Community, Governance, and Culture, including the social and organizational aspects, of ASF projects and the Foundation at large.
```

File: content/blog/cfp-open.md (hunk 2 above)
Comment (RyanSkraba) [inline content/blog/cfp-open.md]:
```suggestion
Access/use of all information presented or referenced must be unencumbered by by patent or other licensing restrictions. 
```

**Comment by RyanSkraba** [comment]
> > Do we need to add the track leader names? I remember that they asked to in the mailing list.
> I don't think for this public facing page. We do need to track that for our in-house communication.

I agree -- I don't know if it would affect "blind" reviews to have the track lead public so early, but I don't think it's strictly necessary yet!

**Pull request merged**